_posts/2022-02-01-attribution.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Attribution"
+date:   2022-02-01 
+author: Agnese Beltramo
+category: Open Science
+tags: attribution
+---
+
+<div style="text-align: justify">
+More and more often, contemporary research is conducted by a number of young and senior researchers actively collaborating and contributing to a common project, answering an increasingly multidisplinary range of questions. 
+
+To ensure all the people involved in the research process are rightfully acknolwegded, the [Contributor Roles Taxonomy, known as CRediT author statement][1] was created. The CRediT taxonomy clearlry provides definitions for a wide range of roles to describe the personal contributions that a researcher might bring to a collaborative project. It also aims at providing a definition for what authorship should mean and acknowledge, in the case of scientific research, by broadening the concept to include all intellectual and practical contributors to the final output. 
```

Comment:
Typo in "clearly"

Reply:
The taxonomy and author statement are two separate concepts.  The taxonomy is a vocabulary with defined meanings for each contributor role.  The author statement then uses this vocabulary.